src/types.js

```diff
@@ -1053,3 +1054,10 @@ export type UtilsType = {|
   getTypeFromName: string => void | typeof Value,
   describeValue: Value => string,
 |};
+
+export type DebuggerConfigArguments = {
+  diagnosticSeverity?: Severity,
+  sourcemaps?: Array<SourceFile>,
+  sourcemapDirectoryRoot?: string,
+  debugChannel?: DebugChannel,
```

Comment:
Moved DebuggerConfigArguments here to address cycle issue. Also moved DebugChannel in here to consolidate all debugger related arguments into one wrapper